django_fsm_log/migrations/0003_statelog_description.py

```diff
@@ -15,6 +15,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='statelog',
             name='description',
-            field=models.TextField(blank=True, null=True),
+            field=models.TextField(blank=True),
```

Comment:
@ticosax 
Why was this fixup done?  (Asking for https://github.com/gizmag/django-fsm-log/issues/83)

Reply:
I don't remember